### DISCUSSION TITLE
Repetitive activities

### DISCUSSION THREAD

**Discussion by WhiteTiger-IT** [kimai/kimai#2303] [Questions & Answers]
How can I use repetitive activities without having to write them down every time?
Meetings and phone calls are made several times, with different customers and for different projects.
It is important to be able to report the time spent on telephone calls, for example, with a customer or on a single project; but also the total time spent on phone calls with all customers and on all projects.

**Comment by kevinpapst** [comment]
You can duplicate the task, you can restart a task, you can drag & drop them in the calendar.
It is preferable if you include ideas for the feature you think of and also what you did and why it did not work out for you.
Improving a software works much better, if it is a bi-directional process.

**Reply by WhiteTiger-IT** [reply]
I think there may be a system-wide list of Activities defined.
For example: Meeting, Phone Calls, EMail, Report, ...
Others could be defined at the Customer level, for example "Meeting with your Suppliers" because I have agreed with that customer that I manage the relationships with its suppliers.
We could consider them as categories.
When I create an activity, I have to associate it with a category.
This way I can create a report with:

- Time spent in Meeting overall.
- Time spent meeting with individual customers.
- Time spent in meetings on individual projects.
- Time spent for a Customer in "Meeting with your Suppliers"
